src/components/Tag/Tag.module.css

```diff
@@ -45,6 +44,7 @@
 
 /**
  * Color variants.
+ * TODO(next-major): remove these status colors
```

Comment:
Are these TODO tracked as tickets?

Reply:
Not currently, tho its not a bad idea to create one and add the detail of "delete all the deprecated things"

Reply:
Added a tracker to the backlog